src/Microsoft.Extensions.RazorViews.Sources/AttributeValue.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.RazorViews
```

Comment:
FYI: I will update the packages.csv in Coherence and Coherence-patch for `Microsoft.Extensions.RazorViews.Sources` and `RazorPageGenerator` as no ship